src/app/pages/HomePage/Graph/ParaTimeSelector/index.tsx

```diff
@@ -284,7 +285,7 @@ const ParaTimeSelectorCmp: FC<ParaTimeSelectorProps> = ({
             <HelpScreen setParaTimeStep={setStep} />
           )}
         </ParaTimeSelectorGlobe>
-        {step === ParaTimeSelectorStep.Explore && (
+        {!hasFixedNetwork() && step === ParaTimeSelectorStep.Explore && (
```

Comment:
Showing graph with fixed network/paratime, makes no sense. Homepage should be different in that case, or directly navigate to dashbaord.

Reply:
Yes, homepage is different.

We _don't_ show the graph when we have a fixed layer. We still show it when we have a fixed network, just without the network switch.

You can check out the demos here: 
- [limited to a network](https://e97738f8.oasis-explorer.pages.dev/)
- [limited to a layer](https://84b385ed.oasis-explorer.pages.dev/)
-  [limited to both network and layer](https://4700fb51.oasis-explorer.pages.dev/)

Reply:
So the only (limited) case when we still want to show the graph is when we are limited to a network, but layer selection is free. In that case, we can still show the selector graph, just hide the mainnet/testnet switch. And this hiding is the code change here. So I think we are good here.